### ISSUE TITLE
CRX_REQUIRED_PROOF_MISSING while installing a theme CRX

### ISSUE THREAD

**Issue by smartfonreddit** [brave/brave-browser#5524]
## Description 
Installing a CRX theme file in brave:extensions shows error CRX_REQUIRED_PROOF_MISSING

## Steps to Reproduce
Use https://www.themebeta.com/chrome-theme-creator-online.html to create a theme file. Download the CRX file. Open brave:extensions and drag it there. It won't install.

## Reproduces how often: 
Every time.

## Brave version (brave://version info)

0.67.123 Chromium: 76.0.3809.87 (Official Build) (64-bit)

## Miscellaneous Information:
https://github.com/manojVivek/medium-unlimited/issues/14

**Comment by smartfonreddit** [comment]
I was somehow able to consistently install the theme without errors by drag-and-dropping it into the Extensions page. The error only happens under certain circumstances.

**Comment by kjozwiak** [comment]
@smartfonreddit would you be able to isolate a particular case where it doesn't work and update the original issue with more accurate steps? This way a developer will have an easier time reproducing and fixing the issue.

**Comment by neveratdennys** [comment]
@smartfonreddit I am also having this issue and it's been a while. Cannot install CRX theme packs directly from theme creator. My particular case is that I cannot install anything on ThemeBeta. My own and other people's custom themes. The official theme store works fine though. Not sure how this can be solved.

**Comment by neveratdennys** [comment]
It seems like only CRX3 is accepted as the extension and theme packages. ThemeBeta's packages looks like they are not in the recently updated format. [https://stackoverflow.com/questions/56930454/chrome-extension-throws-crx-file-error-crx-requird-proof-missing](url)
Hope some kind of fix comes through from Google though. The alternatives are a lot of trouble to go through.

**Comment by EnRyuuX7** [comment]
Had the same problem, did some experiment and now the theme works. 
What I did:
Created a theme and then downloaded the zip and tried to install it by "Drag And Drop" in the extension page which crashed brave
Then made a very simple theme( went to the home page > added a random image > generated colors > "pack and install" ) then went to the downloads folder and Drag dropped the .crx file into the extensions page (chrome://extensions)
It starts to work after this if you can see an "Add theme" message.
Now you can create your theme and instead of going to the 3rd tab to pack and install click the Pack and install on the main page (the page where you see upload an image and generate colors" 
Worked fine for me .
hope this helps.
![image](https://user-images.githubusercontent.com/33335446/65960709-c7be7d80-e474-11e9-8f8c-cda079bf5d4c.png)

**Comment by kurei-ayx** [comment]
I was finally able to install my extension after the .crx keeps failing me.

I made a simple theme (similar to what EnRyuuX7 has done) then instead of "Pack and Install" I went to the tab that says "Pack" and clicked "Pack and Download Zip File" 

I extracted the contents of the zip file (which had a folder entitled "images" and a manifest.json file) > so then I have 2 items, the folder and the manifest, i then put it in a folder (name and location doesn't matter)

You can do either, it works anyways:
A.) Go to chrome://extensions > click on Developer mode > click Load Unpacked > select the folder you just made that consists of the images folder and the manifest > theme installed!

B.) Go to chrome://extensions > click on Developer mode > click Pack extension > browse for extension root directory > select the folder you just made that consists of the images folder and the manifest > there should be a .pem file downloaded > click & drag the .pem file to the extensions page > theme installed!

![Screenshot 2021-05-15 164910](https://user-images.githubusercontent.com/84224756/118354406-832a1c00-b59d-11eb-9c9f-1b725a05fe9e.png)

**Comment by TheBalatron** [comment]
I figured out what was wrong with it. It has to do with the frames. I tried Creating mine using the tabs and got the same error messages over and over. Trying to drag n drop the .crx file directly into extensions and trying to drag the folder created from zip wouldn't read and dropping the zip archive directly would just re download it. 

But then i noticed in dev mode top left theres a few options one says load unpacked tried it didnt work. The other says pack extension and the third is update. from reading what neveratdennys said i assumed its outdated and that one of those options would work. out of curiosity hoping that chromes tools will magically fix it for me i combined that with what kuerl-ayx said. So without settling for the simple creator i was able to unzip my archive to a folder then using googles pack extension option browsed to the folders location and selected it. i then hit pack and it said error and showed a message highlighting specifically the frame.jpg so i went back into themebeta removed the frame only, tried it again and boom it worked. then from there used load unpacked again and there it was. The main issue i have is that it only modifies the new tab page. i really want to modify the main google home search page. any advice on that?
